src/app/public/modules/inline-delete/inline-delete.component.ts

```diff
@@ -121,6 +121,13 @@ export class SkyInlineDeleteComponent implements OnDestroy, OnInit {
     this.adapterService.clearListeners();
   }
 
+  /**
+   * @internal
+   */
+  public onCancelClick(): void {
+    this.animationState = 'hidden';
```

Comment:
See prior comment; this was removed from the template to avoid putting logic in the HTML.